manifests/pagerduty.pp

```diff
@@ -21,7 +21,8 @@
     type    => 'pipe',
     source  => 'puppet:///modules/sensu_handlers/pagerduty.rb',
     config  => {
-      teams => $teams,
+      teams       => $teams,
+      environment => $environment,
```

Comment:
`$environment` is reserved
https://docs.puppet.com/puppet/latest/reference/lang_reserved.html